types/node/vm.d.ts

```diff
@@ -1,7 +1,10 @@
 declare module "vm" {
+    import TypedArray = NodeJS.TypedArray;
```

Comment:
Hu, TIL you can import from namespaces!

Anyhow, this is not consistent to the rest of the code base, please use the namespace directly for now.

Reply:
This line has been removed.

Reply:
Done